components/mixins/experience-event/industry-verticals/experienceevent-autopay-enrollment-details.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-autopay-enrollment-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Autopay Enrollment Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["telecom"]
```

Comment:
I think, auto pay enrollment applies to multiple industries.

Reply:
I expanded the tags for the industries on it.